addons/im_livechat/static/src/models/thread/thread.js

```diff
@@ -17,12 +17,35 @@ registerClassPatchModel('mail.thread', 'im_livechat/static/src/models/thread/thr
      */
     convertData(data) {
         const data2 = this._super(data);
+        if ('last_message_id' in data) {
+            if (!data2.messagesAsServerChannel) {
+                data2.messagesAsServerChannel = [];
+            }
+            data2.messagesAsServerChannel.push(['insert', { id: data.last_message_id }]);
+        }
```

Comment:
This is a bit annoying that we have to update ultra-specific fields in order to manipulate other fields that feel much more natural...
Wouldn't change anything in this PR, just a reminder that even if computed fields is an improvement over none, it's still cumbersome to manage `x2many` relations containing groups...